cli/dcoscli/package/main.py

```diff
@@ -366,9 +365,6 @@ def _install(package_name, package_version, options_path, app_id, cli, app,
 
     if app and pkg.has_mustache_definition():
 
-        # render options before start installation
-        options = pkg.options(user_options)
```

Comment:
We did this explicitly because we want to validate user options before we try installing. That way a user doesn't see the preInstallNotes and then a 'oops bad options' message.